src/restconfig/src/main/java/org/geoserver/rest/catalog/WMSLayerController.java

```diff
@@ -84,19 +84,23 @@ public Object layersGet(
             @PathVariable(required = false) String storeName,
             @RequestParam(required = false, defaultValue = "false") boolean quietOnNotFound,
             @RequestParam(required = false, defaultValue = "configured") String list) {
+        Object result;
         switch (list) {
             case "available":
-                LOGGER.fine(() -> logMessage("GET available WMS layers from ", workspaceName, storeName, null));
-                return new AvailableResources(
+                result = new AvailableResources(
                         getAvailableLayersInternal(workspaceName, storeName, quietOnNotFound), "wmsLayerName");
-            case "configured":
-                LOGGER.fine(() -> logMessage("GET configured WMS layers from ", workspaceName, storeName, null));
+                break;
 
-                return wrapList(
+            case "configured":
+                result = wrapList(
                         getConfiguredLayersInternal(workspaceName, storeName, quietOnNotFound), WMSLayerInfo.class);
+                break;
             default:
                 throw new RestException("Unknown list type " + list, HttpStatus.NOT_IMPLEMENTED);
         }
+        String message = "GET " + list +  " WMS layers from ";
```

Comment:
This can be moved into the logMessage call below to save a string

Reply:
Fixed